dmutils/forms/fields.py

```diff
@@ -44,9 +44,7 @@
 
 
 class DMBooleanField(DMFieldMixin, wtforms.fields.BooleanField):
-    widget = DMCheckboxInput(hide_question=True)
-
-    type = "checkbox"
+    widget = DMCheckboxInput(type="checkbox", hide_question=True)
```

Comment:
I might be missing something obvious here, but why did the custom type previously work for the `DMRadioInput` and not `DMCheckboxInput`?

Reply:
Fixed now